lib/drivers/bayer/bayerContourNext.js

```diff
@@ -397,12 +420,6 @@ module.exports = function (config) {
 
             var packetHead = struct.unpack(packet, 0, '3Z2b', ['HEADER', 'SIZE', 'BYTE1']);
 
-            if(packetHead['HEADER'] !== MAGIC_HEADER){
```

Comment:
Does the Link One have a different magic header? If so, should we set a different MAGIC_HEADER for that model?
Alternatively, if we're not checking the contents of the header at all any more, that we get rid of MAGIC_HEADER in favour of HEADER_SIZE = 3, and removing all references to MAGIC_HEADER.

Reply:
Funnily enough there never was a MAGIC_HEADER in the spec, as it was supposed to contain the report ID, host ID and device ID. All meters prior to the BCN One reported these as ABC, so it seems it has finally been changed to match the spec. These values are not of use to us, so I will follow your alternative suggestion.